deployer/app-deployer/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM golang:1.19-buster as builder
+
+ARG APP_NAME=app-deployer
+ARG BUILD_VERSION
+WORKDIR /app
+
+COPY go.mod ./
+COPY go.sum ./
+RUN go mod download
+
+COPY pkg/ ./pkg
+COPY *.go ./
+
+RUN ls -lh /tmp/
+RUN CGO_ENABLED=0 GOOS=linux go build -v -o deployer -ldflags "-X requestHandler.BuildVersion=$BUILD_VERSION"
+
+FROM quay.io/openshift/origin-cli:latest as origin-cli
+FROM quay.io/ishishov/kustomize:0.0.1 as kustomize
```

Comment:
we should not use private images for these tools
please track an issue to remind changing it (if we cant find one, in alternative we can always use "oc apply -k" for the same, with "--dry-run=client -o=..." option to generate the manifests)

Reply:
Will do, thanks,
Tracked in issue #83